addons/l10n_latam_invoice_document/views/account_move_view.xml

```diff
@@ -45,7 +45,7 @@
                     attrs="{'invisible': [('l10n_latam_use_documents', '=', False)], 'required': [('l10n_latam_use_documents', '=', True)], 'readonly': [('posted_before', '=', True)]}"
                     domain="[('id', 'in', l10n_latam_available_document_type_ids)]" options="{'no_open': True, 'no_create': True}"/>
                 <field name="l10n_latam_document_number"
-                    attrs="{'invisible': ['|', ('l10n_latam_use_documents', '=', False), ('l10n_latam_manual_document_number', '=', False), '|', '|', ('l10n_latam_use_documents', '=', False), ('highest_name', '!=', False), ('state', '!=', 'draft')], 'required': ['|', ('l10n_latam_manual_document_number', '=', True), ('highest_name', '=', False)], 'readonly': [('posted_before', '=', True), ('state', '!=', 'draft')]}"/>
+                    attrs="{'invisible': [('l10n_latam_use_documents', '=', False)], 'required': ['|', ('l10n_latam_manual_document_number', '=', True), ('highest_name', '=', False)]}"/>
```

Comment:
@suv-bmya It does need to stay read-only in some cases.  This is quite a rough change